### ISSUE TITLE
Hyperparameters for All Datasets

### ISSUE THREAD

**Issue by Jiahao004** [Zehong-Wang/HGCML#2]
Hi, may I ask for the hyperparameters for all datasets?

I am able to reproduce the results for acm and dblp two datasets, but not for aminer and freebase. I typically use the command 
/ python3 -W ignore main.py --dataset $dataset --gpu 1 
for the experiments, however, for aminer and freebase datasets, it continues give the results like
/Macro-F1: 0.1813 | Micro-F1: 0.5486 | NMI: 0.0078 | ARI: 0.0084
May I ask for your help?

**Comment by Zehong-Wang** [comment]
Hello, we locate the mistake in the preprocessing of the dataset. We update the dataset.py file by removing the node feature normalization in pre-processing. Notice to use Mean as the feature combiner for IMDB, Aminer, FreeBase, and Concat for ACM and DBLP. E.g., 
```
python main.py --dataset aminer --combine mean
```
We do not make much effort on hyper-parameter tuning. The results reported in the paper can be readily achieved (or approached), even using the default hyper-parameters, on the fixed datasets.py.
